src/NzbDrone.Core/HealthCheck/Checks/DownloadClientRootFolderCheck.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Linq;
+using NLog;
+using NzbDrone.Core.Datastore.Events;
+using NzbDrone.Core.Download;
+using NzbDrone.Core.Download.Clients;
+using NzbDrone.Core.RemotePathMappings;
+using NzbDrone.Core.RootFolders;
+using NzbDrone.Core.ThingiProvider.Events;
+
+namespace NzbDrone.Core.HealthCheck.Checks
+{
+    [CheckOn(typeof(ProviderUpdatedEvent<IDownloadClient>))]
+    [CheckOn(typeof(ProviderDeletedEvent<IDownloadClient>))]
+    [CheckOn(typeof(ModelEvent<RootFolder>))]
+    [CheckOn(typeof(ModelEvent<RemotePathMapping>))]
+
+    public class DownloadClientRootFolderCheck : HealthCheckBase, IProvideHealthCheck
+    {
+        private readonly IProvideDownloadClient _downloadClientProvider;
+        private readonly IRootFolderService _rootFolderService;
+        private readonly Logger _logger;
+
+        public DownloadClientRootFolderCheck(IProvideDownloadClient downloadClientProvider,
+                                      IRootFolderService rootFolderService,
+                                      Logger logger)
+        {
+            _downloadClientProvider = downloadClientProvider;
+            _rootFolderService = rootFolderService;
+            _logger = logger;
+        }
+
+        public override HealthCheck Check()
+        {
+            var clients = _downloadClientProvider.GetDownloadClients();
+            var rootFolders = _rootFolderService.All();
+
+            foreach (var client in clients)
+            {
+                try
+                {
+                    var status = client.GetStatus();
+                    var folders = status.OutputRootFolders;
+                    if (folders != null)
+                    {
+                        foreach (var folder in folders)
+                        {
+                            if (rootFolders.Any(r => r.Path == folder.FullPath))
```

Comment:
This should use `PathEquals` so `C:\Test\TV` matches `C:\Test\TV\` and will properly show an error.